.github/workflows/elixir.yml

```
@@ -29,3 +29,5 @@ jobs:
       run: mix deps.get
     - name: Run tests
       run: mix test
+    - name: Check Formatting
```

The capitalization of the name is inconsistent with the other actions. “Formatting” is not a proper name; make it lowercase.

```suggestion
    - name: Check formatting
```

---

thanks